doc/docusaurus/docs/delve-deeper/further-resources/formal-specification.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 10
+---
+
+# Formal Specification
+
+## Plutus Core Language Specification
+
+The formal specification of the Plutus Core language provides a comprehensive mathematical foundation for understanding the semantics and behavior of Plutus smart contracts.
+
+📄 **[Formal Specification of the Plutus Core Language](https://plutus.cardano.intersectmbo.org/resources/plutus-core-spec.pdf)**
```

Comment:
It's worth noting that it is periodically updated

Reply:
Note added at the end of this page.